docs/api-testing/apifctl-cicd-integration.md

```diff
@@ -365,18 +390,17 @@ Identifies the end date of the events you want to see. For use with **[`metrics`
 ### `-tag <tag(s)>`
 <p><small>| STRING |</small></p>
 
-The **[`run-tag`](#run-tag)** command will run all the tests in your Project marked with that tag. Format as a comma-separated list of tags you want to assign to the test. For use with **[`run-tag`](#run-tag)** command only.
+The **[`run-tag`](#run-tag)** command will run all the tests in your Project marked with that tag. For use with the run-tag command only. For use with the **[`run-tag`](#run-tag)** command only. The example below will run all the tests in the Project labeled with the `product` and `production` tags.
```

Comment:
@kimsauce you said two times the same thing ("For use with the run-tag command only. For use with the **[`run-tag`](#run-tag)** command only." ) I guess one is enough :)

Reply:
yikes!! lol good catch.